android/src/main/java/com/dieam/reactnativepushnotification/modules/RNPushNotification.java

```diff
@@ -68,6 +68,11 @@ public void onNewIntent(Intent intent) {
             bundle.putBoolean("foreground", false);
             intent.putExtra("notification", bundle);
             notifyNotification(bundle);
+        }else if ( intent.getExtras() != null ) {
+            Bundle bundle = intent.getExtras();
+            bundle.putBoolean("foreground", false);
+            intent.putExtra("notification", bundle);
+            notifyNotification(bundle);
         }
     }
```

Comment:
Ok, thanks.  Would you mind resolving  the conflicts and I'll look at adding it in.  Also perhaps rename it, the description isn't accurate.  And could you back out the gradle change (and submit it as a separate PR maybe)?